lib/workos/types/provider_enum.rb

```diff
@@ -9,6 +9,7 @@ class Provider < T::Enum
       enums do
         Google = new('GoogleOAuth')
         Microsoft = new('MicrosoftOAuth')
+        AuthKit = new('authkit')
```

Comment:
Technically `authkit` is only valid for `/user_management/authorize`, but this `Provider` is type is shared with `/sso/authorize`.

So I think we probably shouldn't reuse this type for the new method.

Reply:
That's right, let's create a new one - Maybe `UserManagementProvider`?

Reply:
Sounds good! I might even suggest we put it in the new `WorkOS::UserManagement` module, so `WorkOS::UserManagement::Types::Provider`.

Reply:
@mthadley I've updated it on https://github.com/workos/workos-ruby/pull/236/commits/f6d7dc289519be268f3e1a223f5c13c21f46073d

Let me know if it looks good!